ckanext/qdes_schema/templates/scheming/form_snippets/qdes_multi_group.html

```diff
@@ -4,16 +4,13 @@
     <label class="control-label" for="field-{{ field.field_name }}">{{ field.label }}</label>
     {% if errors[field.field_name] and errors[field.field_name] is iterable %}<span class="error-block">{{ errors[field.field_name]|join(', ') }}</span>{% endif %}
     <div data-repeater-list="{{ field.field_name }}">
-
-        {% set values = h.get_multi_textarea_values(data[field.field_name]) or [] %}
-        {% set count = values['count']|int if values['count']|int > 0 else 1 %}
-        {% for row in range(count)  %}
+        {% for group_data in data[field.field_name] or ['']  %}
```

Comment:
@MarkCalvert I had to change the `[]` to `['']` in order to get the repeatable related resource input fields to display -- I think this is inline with what was previously there, ie.

```
{% set count = values['count']|int if values['count']|int > 0 else 1 %}
```